README.md

```diff
@@ -90,7 +90,13 @@ the value of `getParsedBody()` will be used by default. To disable this behavior
 ```php
 $data = JsonData::fromRequest($request, useParsedBody: false);
 ```
-
+When using `getParsedBody()` remember that most `ServerRequestInterface` objects will decode the request body 
+with `assoc: true`, producing an array. If you wish to have requests decoded as objects, you must set `associative` to false.
+If you're going to set `useParsedBody` to true, and you need an object returned, set `associative` false but keep in mind 
+that an exception will be thrown unless `getParsedBody()` is configured to return an object instead of an array:
+```php
+$data = JsonData::fromRequest($request, useParsedBody: true, associative: false);
```

Comment:
Maybe it would make more sense to have `useParsedBody: true, isObject: true` ? Also this whole thing may not be needed if we had `KeyedDataFactory`?